lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -324,8 +327,8 @@ def decrement_positions_on_higher_items(position)
 
         # This has the effect of moving all the lower items up one.
         def decrement_positions_on_lower_items(position=nil)
-          return unless in_list?
-          position ||= send(position_column).to_i
+          position ||= send(position_column)
+          return unless position.present?
```

Comment:
We use `send(position_column)`, or rather `in_list?` because the position column isn't necessarily called `position`.

Reply:
That's a local variable

Reply:
My bad.

Reply:
I've done some refactoring on `master` and this method now reads:

```
        def decrement_positions_on_lower_items(position=current_position)
          return unless in_list?

          if sequential_updates?
            acts_as_list_list.where("#{quoted_position_column_with_table_name} > ?", position).reorder(acts_as_list_order_argument(:asc)).decrement_sequentially
          else
            acts_as_list_list.where("#{quoted_position_column_with_table_name} > ?", position).decrement_all
          end
        end
```

This should make your changes unnecessary. Let me know though.